src/components/PackedItem/PackedItem.css

```
@@ -1,34 +1,229 @@
-.item--packed {
-  background-color: #454ade;
+.pItem {
+  background-color: white;
   box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.2),
     0 1.5px 3.33px 0 rgba(0, 0, 0, 0.19);
-  width: 60px;
-  height: 102px;
 }
 
-.item--icon {
-  text-align: right;
+.pItem--head {
+  background-color: var(--bundleBlue);
+  border-top-left-radius: var(--br-9);
+  border-top-right-radius: var(--br-9);
+  overflow: scroll;
 }
 
-.item--icon--filler {
-  height: 24px;
+.pItem--buttons {
+  margin-top: 79%;
 }
 
-.item--icon--packed {
-  font-size: 12px;
-  color: white;
+.pItem--button-main {
+  background-color: rgba(0, 0, 0, 0);
+  border: 1px solid rgba(0, 0, 0, 0);
 }
 
-.item--icon--size {
-  font-size: 12px;
-  color: #454ade;
+.pItem--button-right {
+  background-color: white;
+  border-left: 1px solid #4996c9;
+  border-right: 1px solid #4996c9;
+  border-top: 1px solid #4996c9;
+  border-bottom: 1px solid #4996c9;
+  border-radius: 1rem;
+  color: var(--bundleBlue);
 }
 
-.item--image--size {
-  width: 40px;
+.pItem--button-right:hover {
+  background-color: #39b225;
+  border-left: 1px solid #39b225;
+  border-right: 1px solid #39b225;
+  border-top: 1px solid #39b225;
+  border-bottom: 1px solid #39b225;
+  border-top-right-radius: 1rem;
+  border-bottom-right-radius: 1rem;
+  transition: 0.3s;
+  color: var(--white);
+}
+
+.pItem--button-rightDelete {
+  background-color: white;
+  border-left: 1px solid #4996c9;
+  border-right: 1px solid #4996c9;
+  border-top: 1px solid #4996c9;
+  border-bottom: 1px solid #4996c9;
```

```suggestion
  border: 1px solid #4996c9;
```

You don't have to manually specify all sides' borders if they're the same. :wink:

---

Good catch! Idk how I overlooked this, thanks!